src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -505,6 +512,69 @@ export default defineConfig({
 This assumes your integration definition is 1) a `default` export and 2) a function. Ensure this is true before adding the `astro-integration` keyword!
 :::
 
+### `AstroIntegrationLogger`
+
+An instance of the Astro logger, useful to write logs. This logger uses the same [log level](/en/reference/cli-reference/#--verbose)
+configured via CLI.
+
+**Methods available** to write to terminal:
+- `logger.info("Messsage")`;
+- `logger.warn("Messsage")`;
+- `logger.error("Messsage")`;
+- `logger.debug("Messsage")`;
+
+All the messages are prepended with a label that has the same value of the integration.
+
+```ts title="integration.ts"
+import type { AstroIntegration } from "astro";
+export function formatIntegration(): AstroIntegration {
+    return {
+        name: "astro-format",
+        hooks: {
+            "astro:build:done": (options, { logger }) => {
+                // do something
+                logger.info("Integration ready.");
+            }
+        }
+    }
+}
+```
+
+The example above will log something similar to the following:
+
+```shell
+[astro-format] Integration ready.
+```
+
+If you need to log some messages that with a different label, you can a special method called `.fork`:
+
+```ts title="integration.ts"
```

Comment:
```suggestion
```ts title="integration.ts" ".fork"
```
Let's highlight that method!

Reply:
Oh wow I wasn't aware of this, thank you!